src/useEditor.ts

```diff
@@ -108,12 +108,13 @@ export const useEditor = (externalProps: ExternalProps) => {
         (!trackState || trackState.status !== TrackChangesStatus.viewSnapshots)
       ) {
         const sendable = sendableSteps(nextState)
+
         if (sendable) {
           collabProvider.sendSteps(
             sendable.version,
             sendable.steps,
             sendable.clientID,
-            !tr.isGeneric
+            false
```

Comment:
are you sure about that?

Reply:
with that we are going to lose changes for some cases, or I'm missing something about that!

Reply:
It will stop disabling debounce for non generic transactions. Passing through here would cause debounce to be flushed. That would cause too frequent requests to man-api while typing in affiliation form, because each time user types there it updates attributes and goes directly to BE.